WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -129,7 +130,7 @@ final class CardReaderSettingsViewController: UIViewController {
 
     private func addFoundReaderModal() {
         // TODO Use FancyAlert instead
-        let foundReaderName = self.viewmodel.foundReader?.serialNumber ?? ""
+        let foundReaderName = self.viewmodel.foundReader?.serial ?? ""
```

Comment:
Just a though, not a suggestion: this logic (returning the serial or an empty string/placeholder) could be pushed down to the view model?

Reply:
OMG absolutely!

Reply:
So, I took this suggestion a step further, since I'll need these strings elsewhere too (e.g. the fancy alerts) and I now have CardReaderViewModels for connected and found readers created by CardReaderSettingsViewModel - see changes in https://github.com/woocommerce/woocommerce-ios/pull/3784/commits/b7384077caaf1454e3c5bd58550704a402817399

Thoughts? I think it is much cleaner this way.